### DISCUSSION TITLE
course/starbook/learn-word-set-level

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#139] [General]
# course/starbook/learn-word-set-level

通过GoFrame框架实现设置单词掌握程度功能。使用PATCH方法修改ProficiencyLevel字段，引入参数验证确保等级在1到5之间。设计的API路由风格遵循资源层级关系，推荐使用words/{id}/level形式。通过数据库更新完成等级设置，提供接口测试示例以验证功能的正确性。

https://goframe.org/course/starbook/learn-word-set-level

**Comment by nilsir** [comment]
curl 请求上的参数错误, proficiency_level应该为level
